fdbclient/ServerKnobs.h

```diff
@@ -412,6 +414,11 @@ class ServerKnobs : public KnobsImpl<ServerKnobs> {
 	                                             // CC_MAX_HEALTH_RECOVERY_COUNT within
 	                                             // CC_TRACKING_HEALTH_RECOVERY_INTERVAL.
 	int CC_MAX_HEALTH_RECOVERY_COUNT;
+	bool CC_HEALTH_TRIGGER_FAILOVER; // Whether to enable health triggered failover in CC.
```

Comment:
Is health triggered failover the same as gray failure?

Reply:
Yeah, grey failure is included in the failure triggered due to health.